docs/pages/usage/peering.md

```diff
@@ -1,31 +1,87 @@
 ---
-title: Peering
-weight: 4
+title: Peer with a cluster
+weight: 1
 ---
 
 ## Overview
 
-The last step to perform to join two clusters is peering. It allows establishing a connection between the two 
-clusters and to exchange resources.
+In Liqo, peering establishes an administrative connection between two clusters and enables the resource sharing across them.
+It is worth noticing that peering is unidirectional. 
+This implies that resources can be shared only from a cluster to another and not the vice-versa. Obviously, it can be optionally be enabled bi-directionally, enabling a two-way resource sharing.
```

Comment:
```suggestion
This implies that resources can be shared only from a cluster to another and not the vice-versa.
Obviously, it can be optionally be enabled bi-directionally, enabling a two-way resource sharing.
```